internal/cli/format.go

```diff
@@ -0,0 +1,61 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/deref/transcript/internal/core"
+	"github.com/natefinch/atomic"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	formatCmd.Flags().BoolVarP(&formatFlags.DryRun, "dry-run", "n", false, "dry run")
```

Comment:
I don't think dry-run makes sense for a format command. Instead, I think we'd rather format stdin to stdout and then if given arguments, format those in-place.